requirements-dev.txt

```diff
@@ -5,7 +5,7 @@ click==7.0
 coverage==5.3
 cycler==0.10.0
 decorator==4.4.2
-esse==2021.09.09
+esse==2022.11.10.post0
```

Comment:
What is that?

Reply:
OK, it's how the `-0` is added on pypi, I see

Reply:
I took the version number as listed on pypi: https://pypi.org/project/esse/#history Should I use just `2022.11.10` ?